### ISSUE TITLE
Latex axes label issue

### ISSUE THREAD

**Issue by guenp** [plotly/plotly.py#105]
ylabel('R$_{xx,ac}$')
worked in matplotlib, didn't work in plotly

**Comment by benvarkey** [comment]
Any ETA on this? Thanks.

**Comment by chriddyp** [comment]
@benvarkey - You can combine `LaTeX` with plain text by using the `Text` keyword, i.e.: `'$\\beta_{1c} = 25 \\pm 11 \\text{ km s}^{-1}$'`. See more examples here: https://plot.ly/python/LaTeX/

For now, re-arranging the `$` is out of scope for our matplotlib converter.

**Comment by benvarkey** [comment]
It doesn't render with `plotly.offline.iplot(...)` (shows the raw text), but it renders correctly if exported to **plot.ly**

**Comment by benvarkey** [comment]
Actually, I don't get the rendered text even with pure latex code:
<img width="711" alt="screen shot 2015-12-23 at 12 37 42 am" src="https://cloud.githubusercontent.com/assets/2020374/11973337/9b8ab43c-a90d-11e5-8b31-b11b3b6a2e3f.png">

But, when I export to plot.ly, I get this:
<img width="640" alt="screen shot 2015-12-23 at 12 40 12 am" src="https://cloud.githubusercontent.com/assets/2020374/11973363/d7e1fad0-a90d-11e5-8a67-ff64c314f821.png">

Any ideas? Thanks.

**Comment by stephf0716** [comment]
I am experiencing the same issue and can confirm this similarly occurs when using plotly offline with MATLAB.

**Comment by rethore** [comment]
I have the same issue in offline Jupyter notebooks. When I export the plot to the online web-ui it renders the latex label.

**Comment by Kully** [comment]
Hey guys, someone on the Front support is wondering about an estimation for when this feature will be fixed. Any ideas?

**Comment by ma-blaetke** [comment]
I would also love using Latex in plotly offline mode!

**Comment by flo-compbio** [comment]
Just wanted to point out to people here that [inserting a JavaScript snippet into the notebook](https://github.com/plotly/plotly.py/issues/515#issuecomment-263097892) fixed the issue for me. Thanks @jjaraalm !!!

**Comment by guenp** [comment]
@chriddyp maybe you guys could implement this patch? :heart:

**Comment by chriddyp** [comment]
Opening a PR here to try out some different solutions: https://github.com/plotly/plotly.py/pull/683

**Comment by iliyang** [comment]
Same problem here - LaTeX math labels don't render properly in offline mode (in a Jupyter notebook). Hope a fix is coming soon, thanks!

**Comment by ktran9891** [comment]
Adding my voice, as well:  I would love for this to be fixed.

**Comment by KseniiaBelorustceva** [comment]
Same here :) 
+1 for fix request

**Comment by Wes-Skeeter** [comment]
I' having the same issue as well, its pretty frustrating, and its making me not want to use plotly any more.

**Comment by CodeLionX** [comment]
@Wes-Skeeter have you tried using the fix mentioned by **flo-compbio**, see [here](https://github.com/plotly/plotly.py/issues/515#issuecomment-263097892)

this fixed the problem for me, but it's just a workaround and I would also love to have a fix in _plotly_

**Comment by Wes-Skeeter** [comment]
Thanks, but unfortunately, this didn't fix it for me :/  I just want to put an overbar on some text labels, not sue why it has to be so complicated.  HTML works for things like subscripts, Its frustrating I can't use it for this.

**Comment by CodeLionX** [comment]
I'm using _plotly_ in offline mode and HTML is working just fine for me, MWE:
```python
import plotly.offline as offline
from plotly.graph_objs import *

offline.init_notebook_mode(connected=True)
layout = Layout(title = '<span style="text-decoration: overline">text</span>')
fig = dict(data = [Scatter()], layout=layout)
offline.iplot(fig)
```
produces:
![newplot](https://user-images.githubusercontent.com/10573700/34007713-153acc66-e103-11e7-996b-5573393d4176.png)

**Comment by jdieg0** [comment]
What a pity! LaTeX-Code is still not interpreted in offline plots. So I'll have to go back to Matplotlib until this is fixed.

**Comment by jonmmease** [comment]
The offline latex/MathJax issues are going to be resolved by https://github.com/plotly/plotly.js/pull/2994 and https://github.com/plotly/plotly.py/pull/1169.  Follow along if you're interested!

**Comment by jonmmease** [comment]
Done in #1243 and will be released in version 3.4.0.  LaTeX support requires no API changes for `iplot` or `FigureWidget` support.  An additional `include_mathjax` argument has been added to `plotly.offline.plot` to specify how MathJax should be included in the resulting html file/div.
